config/settings.defaults.coffee

```diff
@@ -37,23 +37,12 @@ module.exports =
 				docsWithHistoryOps: ({project_id}) -> "DocsWithHistoryOps:{#{project_id}}"
 
 		project_history:
-			port: process.env["HISTORY_REDIS_PORT"] or process.env["REDIS_PORT"] or "6379"
-			host: process.env["HISTORY_REDIS_HOST"] or process.env["REDIS_HOST"] or "localhost"
-			password: process.env["HISTORY_REDIS_PASSWORD"] or process.env["REDIS_PASSWORD"] or ""
-			maxRetriesPerRequest: parseInt(process.env['REDIS_MAX_RETRIES_PER_REQUEST'] or "20")
```

Comment:
Is this removed on purpose?

Reply:
Ah yes, better to leave that in. I will remove the keepAlive:100 as we previously removed that in 8a46a2726e651288b9b3269873beeac30154af69.